plugins/binance/binance_cli.go

```diff
@@ -0,0 +1,25 @@
+package binance
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func BinanceCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Binance CLI", 
+		Runs:      []string{"binance-cli"},
+		DocsURL:   sdk.URL("https://binance.com/docs/cli"), 
+		NeedsAuth: needsauth.IfAll(
```

Comment:
Do we know if all commands need authentication, or is just "account" subcommands?

Reply:
arun, I think almost all commands need authentication